### PR TITLE
Improves explaination of modem presets

### PR DESCRIPTION
**Pull request by charminULTRA**
Had to re-do this completely after figuring out how to submit this properly.

### CODE REVIEW

**Comment by charminULTRA** [comment]
> > Had to re-do this completely after figuring out how to submit this properly.
> 
> That did it!

Thanks for the help! Are you on the Discord?

File: docs/configuration/device-config/lora.mdx [hunk 1]
```diff
@@ -11,6 +11,10 @@ import LoRaRegions from "../../blocks/_lora-regions.mdx";
 
 The LoRa config options are: Region, Modem Preset, Max Hops, Transmit Power, Bandwidth, Spread Factor, Coding Rate, Frequency Offset, Transmit Enabled, Channel Number and Ignore Incoming Array. LoRa config uses an admin message sending a `Config.LoRa` protobuf.
 
+:::note
+In order to communicate fully, devices within a mesh must have identical settings for Region and Modem Preset, or identical custom Modem settings.
```
Comment (jfirwin) [inline docs/configuration/device-config/lora.mdx:15]: Great admonition, it's something I want to convey visually as well down the line.

File: docs/configuration/device-config/lora.mdx [hunk 2]
```diff
@@ -25,7 +29,27 @@ Sets the region for your node. Default is `unset`. As long as this is not set, t
...
+2. `SHORT_SLOW`
+
+3. `MEDIUM_FAST`
+
+4. `MEDIUM_SLOW`
+
+5. `LONG_FAST` (Default)
+
+6. `LONG_SLOW`
+7. `VERY_LONG_SLOW` (Slowest, lowest bandwidth, highest airtime, longest range)
```
Comment (jfirwin) [inline docs/configuration/device-config/lora.mdx:52]: Looks great! One thing we may consider is mentioning device count limits here. But we can do that another day.

**Review by jfirwin — APPROVED** [review]
This looks great! Sorry for the run around with the checks on your previous PRs. Was it trunk that ended up giving you issues? That's a new dependency so I'm all ears for how it's going. I did these review on mobile so it's a little harder to dig around 🙂 thanks for the contribution!

**Pull request merged**